src/components/organisms/AdvancedLending/AdvancedLending.tsx

```diff
@@ -473,18 +474,31 @@ export const AdvancedLending = ({
                         </div>
                     </div>
 
-                    <div className='hidden laptop:block laptop:w-[272px] desktop:w-[300px]'>
-                        {!isTablet && (
-                            <NewOrderBookWidget
-                                orderbook={orderBook}
+                    <div className='hidden laptop:block laptop:h-[473px] laptop:w-[272px] desktop:w-[300px]'>
```

Comment:
why do we need to fix the height?

Reply:
I see that removing it does not affect its children. I will remove this rule